wsl-module/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pip freeze > ./requirements.txt
```

Comment:
This should not be here imo. Once you set up the dockerfile with the neccessary `requirements.txt`, that file is already pushed in the repo, so there is no need to do this command again, since it will be overwritten and the docker image will be built with the wrong requirements.

So this line can be erased. Right?

Reply:
Correct. Adding it to gitignore.